### PR TITLE
[TW-615] Doc governance for public documentation

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good to me!

File: src/pages/docs/publishing-your-api/publishing-your-docs.md [hunk 1]
```diff
@@ -80,11 +80,11 @@ To publish the documentation for a collection:
...
+If you have a [Team Admin role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), you can customize your published documentation with a team logo. To add a logo, [edit your team profile](/docs/administration/team-settings/#editing-your-team-profile) in your [team settings](https://go.postman.co/settings/team/general). Your team logo will replace the Postman logo in your team's published documentation. (It may take several minutes for the new logo to appear.)
 
 ## Sharing your public docs
 
-To share your public documentation, share the published URL with your team members, other users, or the community. If you have forgotten the URL for your published documentation, you get it at any time:
+To share your public documentation, share the published URL with your team members, other users, or the community. If you have forgotten the URL of your published documentation, you get it at any time:
```
Comment (petewitcher) [inline src/pages/docs/publishing-your-api/publishing-your-docs.md:87]: Should it be "you can get it at any time"?

**Review by petewitcher — APPROVED** [review]
LGTM with one suggestion.

File: src/pages/docs/publishing-your-api/publishing-your-docs.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/publishing-your-api/publishing-your-docs.md]: Yes, thanks!

**Review by dhwaneetbhatt — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Review by abhinav-ravi — APPROVED** [review]

**Pull request merged**